CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ endif ()
 # special debug flags
 set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -D_DEBUG")
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -stdlib=libstdc++ -fsanitize=undefined,address,integer,unsigned-integer-overflow -fno-omit-frame-pointer -fsanitize-blacklist=${CMAKE_SOURCE_DIR}/build/clang_sanitize_blacklist.txt")
+    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fsanitize=undefined,address,integer,unsigned-integer-overflow -fno-omit-frame-pointer -fsanitize-blacklist=${CMAKE_SOURCE_DIR}/build/clang_sanitize_blacklist.txt")
```

Comment:
What is the point in removing this? I think I remember that we had build problems with the clang libstdc especially on Mac.

Reply:
Yes that is exactly the problem. On a Mac with clang, the -stdlib=libstdc++ causes the compilation to fail when including the <cstdlib> and the <random> libraries. This was the only way I've found to deal with this issue.

Reply:
Are you bulding the "standard" brew way?

Reply:
Actually I don't know what the standard brew way is :) I'm using `cmake` the following way:
```bash
mkdir build-release
cd build-release
CC=clang CXX=clang++ cmake -DCMAKE_BUILD_TYPE=Release ..
make
```

Reply:
I just meant the way you installed the dependent libraries because I think the main point is that all need to use the same cstdlib.